README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Prerequisites
 You wil need MongoDB to run this app. You can download it [here](https://www.mongodb.com/download-center/community). Then in your root .env file, add your mongo uri as MONGO_URI. You can generally find your mongo uri as the port where your MongoDB is running followed by /`Name of the database`
```

Comment:
Might be helpful to add a bit more on mongo setup -- especially given that homebrew removed the mongodb formula and there is so much outdated setup info out there.

Reply:
I feel having a complete setup guide might be a good idea, especially since I haven't used Mongoose and Mongo.

Reply:
I'm not sure how it is with homebrew, but I know in general with mongoose, it sets up your database and models once it's connected. I've only had to install mongodb on my machine, then run the command `mongod` in a terminal. Hopefully this can help? https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/
Once mongod is running, mongoose should be able to connect to it once it has the proper MONGO_URI variable. I forgot to mention that if you have a password the format is like this: `mongodb://admin:password@localhost:27017/team-summer` . 
Robo 3T is also a good GUI for mongodb. You can download it here: https://robomongo.org/download